backend.native/tests/framework/stdlib/stdlib.swift

```diff
@@ -314,9 +314,9 @@ class StdlibTests : TestProvider {
     }
 
     func testSet() throws {
-        let elements = KotlinArray(size: 2) { index in nil }
+        let elements = KotlinArray<AnyObject>(size: 2) { index in nil }
         elements.set(index: 0, value: nil)
-        elements.set(index: 1, value: 42)
+        elements.set(index: 1, value: 42 as NSNumber)
```

Comment:
Is it always required now?

Reply:
Not exactly. Swift doesn't actually have implicit conversion from its integer types to `NSNumber`. There is ad hoc support for literals, e.g.
```swift
let x: NSNumber = 42
```
works.
But in the discussed line of test the expected type is not `NSNumber`, so such a literal conversion doesn't apply.